src/styles/holographic.css

```
@@ -1,856 +1,157 @@
+/* Estilos Holográficos Pulidos para J-Vairyx */
 
-.processor-rings {
-  position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
+:root {
+  --vairyx-glow: #64b5f6;
+  --vairyx-accent: #ba68c8;
+  --vairyx-bg: rgba(15, 15, 25, 0.85);
```

Stylelint (17.2.0)

[error] Expected "0.85" to be "85%" (alpha-value-notation)

(alpha-value-notation)